### ISSUE TITLE
Google Pub/Sub gRPC Unable to connect

### ISSUE THREAD

**Issue by lustefaniak** [akka/alpakka#2232]
### Versions used 

Akka version: 2.6.4
Alpakka version: 2.0.0-RC1

### Expected Behavior

Cloud Pub/Sub subscription should work.

### Actual Behavior

I defined `rootCa` to the certificate which worked fine with `1.x` and `2.0.0-M3`, but when I run my application after upgrading to RC1 I get:

```
com.typesafe.sslconfig.ssl.CompositeCertificateException: No trust manager was able to validate this certificate chain: # of exceptions = 1
```

After enabling tracing I can see there is:

```
exception: checkServerTrusted(chain -> Array(X509Certificate(serialNumber = 90768918e93393a0, subject = CN=invalid2.invalid, OU="No SNI provided; please fix your client.")),authType -> ECDHE_RSA)
```

**Comment by raboof** [comment]
@lustefaniak  I suspect this might be fixed in snapshot `0.8.2+5-c7e7794a` of akka-grpc.

Could you test this by adding `resolvers += Resolver.bintrayRepo("akka", "maven")` to your sbt config and adding an explicit dependency on `"com.lightbend.akka.grpc" % akka-grpc-runtime" %% "0.8.2+5-c7e7794a"`?

**Comment by ennru** [comment]
Fixed with #2259
